src/main/java/gov/nasa/pds/tools/label/SchematronTransformer.java

```diff
@@ -60,13 +61,14 @@ public SchematronTransformer() throws TransformerConfigurationException {
     // Use saxon for schematron (i.e. the XSLT generation).
     System.setProperty("javax.xml.transform.TransformerFactory",
         "net.sf.saxon.TransformerFactoryImpl");
+    DocumentBuilderFactory.newInstance().setNamespaceAware(true);
     TransformerFactory isoFactory = TransformerFactory.newInstance();
     // Set the resolver that will look in the jar for imports
     isoFactory.setURIResolver(new XslURIResolver());
     // Load the isoSchematron stylesheet that will be used to transform each
     // schematron file
     Source isoSchematron = new StreamSource(
-        LabelValidator.class.getResourceAsStream("/schematron/iso_svrl_for_xslt2.xsl"));
+        LabelValidator.class.getResourceAsStream("/schematron/simple_test.xsl"));
```

Comment:
@al-niessner just as a heads up, I think this was left behind in your commits by mistake?

Reply:
Yes. Fixed.